usage/test-support/src/main/java/org/apache/brooklyn/test/performance/MeasurementOptions.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.test.performance;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.io.File;
+import java.util.concurrent.CountDownLatch;
+
+import org.apache.brooklyn.util.time.Duration;
+import org.apache.commons.io.FileUtils;
+
+import com.google.common.annotations.Beta;
+import com.google.common.base.Objects;
+
+/**
+ * For building up a description of what to measure.
+ * 
+ * See {@link PerformanceTestUtils#measure(MeasurementOptions)}.
+ */
+@Beta
+public class MeasurementOptions {
```

Comment:
`MeasurementOptions` is a cryptic name.  `PerformanceTestOptions` ? or `PerformanceTestDescriptor` (as some aren't optional) ?